src/NexusMods.App.UI/Pages/CollectionDownload/CollectionComponents.cs

```diff
@@ -0,0 +1,213 @@
+using JetBrains.Annotations;
+using NexusMods.Abstractions.Jobs;
+using NexusMods.Abstractions.NexusModsLibrary.Models;
+using NexusMods.Abstractions.UI;
+using NexusMods.Abstractions.UI.Extensions;
+using NexusMods.App.UI.Controls;
+using NexusMods.App.UI.Pages.LibraryPage;
+using NexusMods.MnemonicDB.Abstractions.Models;
+using R3;
+
+namespace NexusMods.App.UI.Pages.CollectionDownload;
+using CollectionDownloadEntity = NexusMods.Abstractions.NexusModsLibrary.Models.CollectionDownload;
+
+public static class CollectionColumns
+{
+    [UsedImplicitly]
+    public sealed class Actions : ICompositeColumnDefinition<Actions>
+    {
+        public static int Compare<TKey>(CompositeItemModel<TKey> a, CompositeItemModel<TKey> b) where TKey : notnull
+        {
+            // TODO:
+            return 0;
+        }
```

Comment:
Was this forgotten or did you want to do it in a future pr?

Reply:
Future pr, sorting doesn't work anyways right now.